aos/closed_loop/calcZernikesFromDonutStamps.ipynb

```diff
@@ -8,10 +8,10 @@
     "# CalcZernikes from DonutStamps\n",
```

Comment:
<p>Is RMSE the root mean squared estimate? I'd mention here that the reduction of RMS is due to removing outliers (i.e. to explicitly say that RMS is a measure of scatter, and so a smaller RMS means smaller mean departure  from the OPD, i.e.  Zernike polynomial used for final estimate were closer to the OPD thanks to the sigma clipping). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>